scylla-cql/src/deserialize/value_tests.rs

```diff
@@ -107,8 +119,11 @@ fn test_deserialize_text() {
         deserialize::<&str>(&ColumnType::Native(NativeType::Text), &unicode).unwrap();
     let decoded_text_string =
         deserialize::<String>(&ColumnType::Native(NativeType::Text), &unicode).unwrap();
+    let decoded_text_boxed_str =
+        deserialize::<Box<str>>(&ColumnType::Native(NativeType::Text), &unicode).unwrap();
     assert_eq!(decoded_text_str, UNICODE_TEXT);
     assert_eq!(decoded_text_string, UNICODE_TEXT);
+    assert_eq!(decoded_text_boxed_str.as_ref(), UNICODE_TEXT);
 
```

Comment:
In `test_deserialize_text` you only added tests for boxed str, but not Arced str. Could you also add them?